x-pack/plugins/security_solution/public/common/lib/process_actions/index.ts

```diff
@@ -6,17 +6,26 @@
  */
 
 import type {
-  KillProcessRequestBody,
+  KillOrSuspendProcessRequestBody,
   ResponseActionApiResponse,
 } from '../../../../common/endpoint/types';
 import { KibanaServices } from '../kibana';
-import { KILL_PROCESS_ROUTE } from '../../../../common/endpoint/constants';
+import { KILL_PROCESS_ROUTE, SUSPEND_PROCESS_ROUTE } from '../../../../common/endpoint/constants';
 
 /** Kills a process specified by pid or entity id on a host running Endpoint Security */
 export const killProcess = async (
-  params: KillProcessRequestBody
+  params: KillOrSuspendProcessRequestBody
 ): Promise<ResponseActionApiResponse> => {
   return KibanaServices.get().http.post<ResponseActionApiResponse>(KILL_PROCESS_ROUTE, {
     body: JSON.stringify(params),
   });
 };
+
+/** Suspends a process specified by pid or entity id on a host running Endpoint Security */
+export const suspendProcess = async (
+  params: KillOrSuspendProcessRequestBody
+): Promise<ResponseActionApiResponse> => {
+  return KibanaServices.get().http.post<ResponseActionApiResponse>(SUSPEND_PROCESS_ROUTE, {
+    body: JSON.stringify(params),
+  });
```

Comment:
I missed this earlier, but since we're just returning the promises here instead of waiting, we don't need these functions to be `async`.